build.gradle

```diff
@@ -176,18 +175,21 @@ static def nextBuildNumber(ENV, version_id) {
 						if (number > build_number) {
 							build_number = number
 						}
-					} catch (NumberFormatException ignored) {
-
+					} catch (NumberFormatException e) {
+						throw new RuntimeException(e);
 					}
 				}
 			}
 		} catch (FileNotFoundException e) {
-
+			throw new RuntimeException(e);
```

Comment:
I'm not sure about this. While it makes sense for us now, given that we already have some builds on the maven, this would have thrown when we published our first builds. Basically no metadata file exists, we assume that means no publications exist yet, ergo next build number is 1. On the other hand, we *do* already have plenty of publications, so if no metadata is found, something is definitely wrong.

Reply:
...hmmm this is a perfect "chicken or the egg" case...
I think we should throw this exception but leave a comment behind that shows people that this assumes a metadata file exists, and if it doesn't this fails and in case they don't have a metadata file they should remove this...

Reply:
that sounds good to me